Description: Fix -M example from "-M -20000" to "-M 20000"
 Thanks to Rhalina for noticing it this mistake.
Author: Axel Beckert <abe@debian.org>

Index: autossh/autossh.1
===================================================================
--- autossh.orig/autossh.1	2013-10-17 20:03:06.744569099 +0200
+++ autossh/autossh.1	2013-10-17 20:15:46.249437226 +0200
@@ -160,7 +160,7 @@
 .Nm 
 will send test data on the base monitoring port, and
 receive it back on the port above. For example, if you specify
-.Dq Fl M 20000 ,
+.Dq Fl M\ 20000 ,
 .Nm
 will set up forwards so that it can send data on port
 20000 and receive it back on 20001.
